src/storage-preview/README.md

```diff
@@ -159,22 +159,22 @@ You can find the command mapping from ADLS Gen1 to ADLS Gen2 as follows:
 |:----------------------------------------------:|:----------------------------------------------------:|
 | az dls fs                                      | az storage blob directory                            |
 | az dls fs access                               | az storage blob directory access                     |
-| az dls fs access remove-all                    | NO                                                   |
-| az dls fs access remove-entry                  | NO                                                   |
+| az dls fs access remove-all                    | Not currently supported                              |
```

Comment:
Should be "currently not supported".
"not currently supported" sounds a bit strange.

Reply:
OK... Actually it is what service PM requested. What about N/A?

Reply:
Vote "Currently not supported"